### PR TITLE
Clean up schemas and other stuff...

### PR DESCRIPTION
**Pull request by ivargrimstad**
Signed-off-by: Ivar Grimstad <ivar.grimstad@eclipse-foundation.org>

### CODE REVIEW

**Review by gtudan — APPROVED** [review]
We should also update the quickstart-documentation: https://github.com/eclipse-ee4j/krazo/blob/master/documentation/src/main/asciidoc/_quickstart.adoc

It contains a couple of examples that still have the JCP-Namespace

**Review by chkal — APPROVED** [review]

File: testsuite/src/main/resources/encoding/views/iso-8859-15.xhtml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="de"
       xmlns:f="http://xmlns.jcp.org/jsf/core"
       xmlns:h="http://xmlns.jcp.org/jsf/html">
-    <f:view encoding="ISO-8859-15" contentType="text/html; charset=ISO-8859-15">
```
Comment (chkal) [inline testsuite/src/main/resources/encoding/views/iso-8859-15.xhtml:5]: Looks like for some reason this broke the testsuite:

```
Failed tests: 
  EncodingIT.facelets_should_allow_produces_text_html:111->checkContentType:129 expected:<text/html;charset=[iso-8859-15]> but was:<text/html;charset=[utf-8]>
  EncodingIT.facelets_should_ignore_produces_text_html_with_charset:118->checkContentType:129 expected:<text/html;charset=[iso-8859-15]> but was:<text/html;charset=[utf-8]>
  EncodingIT.facelets_should_use_content_type_view_attribute:104->checkContentType:129 expected:<text/html;charset=[iso-8859-15]> but was:<text/html;charset=[utf-8]>
Tests run: 54, Failures: 3, Errors: 0, Skipped: 4
```

**Review by chkal — CHANGES_REQUESTED** [review]

File: testsuite/src/main/resources/encoding/views/iso-8859-15.xhtml (hunk 1 above)
Comment (ivargrimstad) [reply testsuite/src/main/resources/encoding/views/iso-8859-15.xhtml]: I added the encoding parameter back. Was removed by mistake...

**Review by chkal — APPROVED** [review]
Looks good to me. Too bad that we are hitting the Travis resource limits.

**Pull request merged**